charger/abb.go

```diff
@@ -138,6 +130,13 @@ func (wb *ABB) Status() (api.ChargeStatus, error) {
 	case 4: // State C2: Charging Contact closed, energy delivering
 		return api.StatusC, nil
 	case 5: // Other: Session stopped
+		b, err := wb.conn.ReadHoldingRegisters(abbRegSocketLock, 2)
+		if err != nil {
+			return api.StatusNone, err
+		}
+		if binary.BigEndian.Uint32(b) >= 0x0101 {
```

Comment:
Magische Konstante? Bedeutung?

Reply:
Ja, magische Konstante. Das ist der interne Minimalstrom.
Man darf gar nicht erst versuchen zu verstehen was die da in der FW machen...